.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "seneca",
+  "parser": "babel-eslint",
+  "plugins": [
+    "react"
+  ],
+  "ecmaFeatures": {
+    "jsx": true,
+    "modules": true
+  },
+  "env": {
+    "browser": true,
+    "amd": true,
+    "es6": true,
+    "node": true
+  }
+}
```

Comment:
please remove everything except
`"extends": "seneca"`
That's the only linting rule that we should use